src/algorithm/centroidal.hxx

```diff
@@ -53,14 +53,12 @@ namespace pinocchio
 
       data.mass[i] = Y.mass();
       data.com[i] = Y.mass() * Y.lever();
-      
+
+      data.v[i] = jdata.v();
```

Comment:
The change are not optimal. The Original version is optimal with respect to the Joint API. Can you remove this commit?

Reply:
I tried to keep the same loop as in forwardkinematics.
nevermind. changed.